### ISSUE TITLE
Method to label frames using coordinates from previous GUI labeling

### ISSUE THREAD

**Issue by tomaitagaki** [DeepLabCut/DeepLabCut#343]
Hello,

TL;DR: I have the x,y coordinates of all the labels; how can I use them to label frames in a new project (same extracted frames) without having to use the GUI again.

DLC Version: 2.0.7 (Maybe an older version before 2.0.4 will work?)
System: Windows/dlc-windowsCPU anaconda env + Ubuntu 16.04 LS on Google Cloud

When trying out the new 3D triangulation method, I ran into the issue of not having the body part labels in my two 2D projects named the same (ie. pointernail vs pointernail2, thumbknuckle vs thumbknuckle2) and tried to go back and rename the labels in one of the 2D project config.yaml files to match the other one but I kept running into issues.

So I decided to remake the camera-2 project and name the labels the same as the camera-1 project (ie. pointernail vs pointernail, thumbknuckle vs thumbknuckle). Since I have the x, y coordinates of the labels from the .csv file in the /labeled-data/video-camera-2/ folder, I wanted to use those coordinates instead of relabeling my 15 body parts for 80 frames. The frames I would be labeling would be the same frames I originally labeled using the GUI so the coordinates stored in the CollectedData_Scorer.csv file will work.

Is there a way to use these raw coordinates or should I go and relabel using the GUI...

Thanks!

**Comment by MMathisLab** [comment]
You can edit the labeled data csv file then convert to h5 with deeplabcut.convertcsv2h5
